zap/src/main/java/org/parosproxy/paros/extension/manualrequest/http/impl/HttpPanelSender.java

```diff
@@ -91,7 +100,13 @@ public void handleSendMessage(Message aMessage) throws IllegalArgumentException,
         httpMessage.setRequestingUser(null);
         try {
             final ModeRedirectionValidator redirectionValidator = new ModeRedirectionValidator();
+            boolean useAntiCSRF = getButtonUseCsrf().isSelected();
             boolean followRedirects = getButtonFollowRedirects().isSelected();
+
+            if (useAntiCSRF && extAntiCSRF != null) {
```

Comment:
Worth swapping the operands and inline the `isSelected` check to avoid creating the button when the extension is disabled.